docker/Dockerfile-depl

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:bionic-20190807 AS install_dependencies
 
 RUN apt-get update  && apt-get install -y cmake git build-essential libgtest-dev libssl-dev qtbase5-dev \
     zip libmysqlcppconn-dev libboost1.65-all-dev libmysqlclient-dev uuid-dev libxerces-c-dev qtbase5-dev \
-    libcurl4-openssl-dev libgps-dev libsnmp-dev librdkafka-dev libev-dev libuv-dev libcpprest-dev
+    libcurl4-openssl-dev libgps-dev libsnmp-dev librdkafka-dev libjsoncpp-dev libev-dev libuv-dev libcpprest-dev
```

Comment:
I remember we needed to add this but I can't remember why.

Reply:
convert the MobilityPath,BSM, mobilityOperation to json string

Reply:
That right. Thanks